ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/baselinecaps/v411/provision/cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-upi-baselinecaps-v411-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: ipi-conf-azure
+  - ref: upi-install-azure
+  - ref: enable-qe-catalogsource
```

Comment:
If it is v411 baselinecaps, will olm operators be enabled or not? if not, I guess we do not need enable-qe-catalogsource